File: bddtests/src/main/kotlin/mocking/vision/models/Account.kt
package mocking.vision.models

import javax.xml.bind.annotation.XmlElement

data class Account(@XmlElement(namespace = "urn:vision")
                   var patientId: String,

                   @XmlElement(namespace= "urn:vision",name = "patientNumber")
                   var patientNumber: List<PatientNumber>?,

                   @XmlElement(namespace = "urn:vision")
                   var name: String) {
    constructor() : this("", null, "")
}

File: xamarinintegrationtests/NHSOnline.HttpMocks/Vision/Models/Account.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Vision.Models
{
    internal sealed class Account
    {
        [XmlElement(ElementName = "patientId", Namespace = "urn:vision")]
        public string? PatientId { get; set; }

        [XmlElement(ElementName = "name", Namespace = "urn:vision")]
        public string? Name { get; set; }

        [XmlElement(ElementName = "patientNumber", Namespace = "urn:vision")]
        public List<PatientNumber> PatientNumbers { get; } = new List<PatientNumber>();
    }
}